### PR TITLE
add basic REST test for geohash_grid

### PR DESCRIPTION
**Pull request by talevy**
this PR is a part of the general initiative to get more REST coverage in aggregations: #26220

### CODE REVIEW

File: rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/280_geohash_grid.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+---
+"Basic test":
+  - do:
+      bulk:
+        refresh: true
+        body:
+          - index:
+              _index: test_1
+              _type:  _doc
```
Comment (talevy) [inline rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/280_geohash_grid.yml]: ah. mental note. need to remove mention of type as is done in https://github.com/elastic/elasticsearch/pull/37611

**Comment by colings86** [comment]
Please could you merge this after #37611 to avoid us having to do another iteration on that large PR?

**Comment by colings86** [comment]
https://github.com/elastic/elasticsearch/pull/37611 is now merged

**Review by polyfractal — APPROVED** [review]
:+1: thanks!

If you're feeling extra motivated we could add a test for illegal `precision` values, to verify the exception

**Pull request merged**